### DISCUSSION TITLE
How to chnage the blog page to something else?

### DISCUSSION THREAD

**Discussion by muditbhargava66** [alshedivat/al-folio#2483] [Q&A] [answered]
I write my blogs and it is hosted by quartz under the naming of `notes` like `xyz.github.io/notes` for my gh-pages. How can i add that page directly in the folio website and where to comment `blogs` section so it won't give any error?

**Comment by george-gca** [comment]
> How can i add that page directly in the folio website

Simply link to it like an external website, with the full url.

> and where to comment blogs section so it won't give any error?

You can simple remove `_pages/blog.md` and also the `_posts/` directory.

**Comment by muditbhargava66** [comment]
> Simply link to it like an external website, with the full URL.

In which code file should the link be placed exactly?

**Comment by george-gca** [comment, marked as answer]
Ok now I get it. You'll need to do a change in [_includes/header.liquid](https://github.com/alshedivat/al-folio/blob/5beffc317916a69fd8f9368f12bf7dbbf06a1a38/_includes/header.liquid#L98), more specifically do something like this part:

```html
<li class="nav-item {% if page.url contains p.permalink %}active{% endif %}">
  {% if p.permalink contains '/blog/' %}{% assign url = '/blog/' %} {% else %}{% assign url = p.url %}{% endif %}
  <a class="nav-link" href="{{ url | relative_url }}">
    {{- p.title }}
    {% if page.url contains p.url %}
      <span class="sr-only">(current)</span>
    {% endif %}
  </a>
</li>
```

But simply set the `href` to your `xyz.github.io/notes`. Just note that this part of the code is responsible for creating all the pages in the header in the order they were set, so if you want the blog to be in a specific order in the header you have to take that into account.
